ui/imports/shared/controls/ImportKeypairInfo.qml

```diff
@@ -0,0 +1,55 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Core.Theme 0.1
+
+import utils 1.0
+
+Rectangle {
+    id: root
+
+    radius: Style.current.radius
```

Comment:
this after property declarations?

Reply:
Technically it's the same, but will change this to have custom props first.